.changes/v3.9.0/988-improvements.md

```diff
@@ -0,0 +1 @@
+* `vcd_org_vdc` includes a property `enable_distributed_firewall` to enable or disable a NSX-V distributed firewall [GH-988]
```

Comment:
The name of the property has became outdated.

Reply:
done